docs/community-ee/community-ee-announcement.md

```diff
@@ -115,4 +96,40 @@ Ansible Community Execution Environment Minimal <ansible-core-version-minimal-ee
 Get the details of both the images [here](<link to the forum post>).
 
 On behalf of the Ansible community, thank you and happy automating!
+
 ```
+
+
+## Bullhorn Announcement
+
+
+We’re happy to announce the release of the Ansible Community Execution Environment Minimal <version> (Latest) and Ansible Community Execution Environment Base <version> are here.
+
+Read the whole announcement [here](link to forum announcement).
```

Comment:
```suggestion
Read the [whole announcement](link to forum announcement).
```

Reply:
Just to point out that this is a best practice for accessibility because it gives more context. As a link, `[here]` is ambiguous and does not provide any information or context about the url to which it points.

Loads of info available at: https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context